rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+- name: create namespace
+  openshift_v1_project:
+    name: '{{ namespace }}'
+
+- name: create volumes
+  k8s_v1_persistent_volume_claim:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    access_modes:
+      - ReadWriteOnce
+    resources_requests:
+      storage: 1Gi
+
+- name: create service
+  k8s_v1_service:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    labels:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    selector:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    ports:
+    - name: port-5432
+      port: 5432
+      protocol: TCP
+      target_port: 5432
+  register: postgres_service
+
+- name: create deployment config
+  openshift_v1_deployment_config:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    labels:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    replicas: 1
+    selector:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    strategy_type: Rolling
+    strategy_rolling_params:
+      interval_seconds: 1
+      max_surge: 25%
+      max_unavailable: 25%
+      timeout_seconds: 600
+      update_period_seconds: 1
+    spec_template_metadata_labels:
+      app: rhscl-postgresql-apb
+      service: postgresql
+    containers:
+    - env:
+      - name: POSTGRESQL_PASSWORD
+        value: '{{ postgresql_password }}'
+      - name: POSTGRESQL_USER
+        value: '{{ postgresql_user }}'
+      - name: POSTGRESQL_DATABASE
+        value: '{{ postgresql_database }}'
+      image: registry.access.redhat.com/rhscl/{{ image }}
+      image_pull_policy: IfNotPresent
+      name: postgresql
+      ports:
+      - container_port: 5432
+        protocol: TCP
+      resources: {}
+      security_context: {}
+      termination_message_path: /dev/termination-log
+      volume_mounts:
+      - mount_path: /var/lib/pgsql/data
+        name: postgresql
+      working_dir: /
+    dns_policy: ClusterFirst
+    restart_policy: Always
+    termination_grace_period_seconds: 30
+    volumes:
+    - name: postgresql
+      persistent_volume_claim:
+        claim_name: postgresql
+      test: false
```

Comment:
volumes is in line with everything under openshift_v1_deployment_config. I think it's right? I don't know I don't have much experience with the yaml for creating these resources, but I do see in the UI that the voume is mounted under /var/lib/pgsql/data or whatever it is.